src/lib.rs

```diff
@@ -3334,6 +3334,21 @@ impl Build {
                     // Use the GNU-variant to match other Unix systems.
                     name = format!("g{}", tool).into();
                     self.cmd(&name)
+                } else if target.os == "vxworks" {
+                    name = format!("wr-{}", tool).into();
+                    self.cmd(&name)
+                } else if target.os == "nto" {
+                    name = match target.arch {
+                        "i586" | "x86" => format!("ntox86-{}", tool).into(),
+                        "aarch64" | "x86_64" => format!("nto{}-{}", target.arch, tool).into(),
+                        _ => {
+                            return Err(Error::new(
+                                ErrorKind::InvalidTarget,
+                                format!("Unknown architecture for Neutrino QNX: {}", target.arch),
+                            ))
+                        }
+                    };
+                    self.cmd(&name)
```

Comment:
I know it's not really done elsewhere (yet), but would you mind adding documentation links to `wr-ar`, `ntoaarch64-ar`, etc.? I.e. documentation on where these names come from - would make it easier to maintain.

Reply:
I couldn't find any documentation for `wr-ar`. I added one for `nto`, but I'm not sure how useful it is.